heat/sparse/manipulations.py

```diff
@@ -0,0 +1,75 @@
+"""
+Manipulation operations for (potentially distributed) `Dcsr_matrix`.
+"""
+from __future__ import annotations
+
+from heat.sparse.dcsr_matrix import Dcsr_matrix
+
+from ..core.dndarray import DNDarray
+from ..core.factories import empty
+
+__all__ = [
+    "todense",
+]
+
+
+def todense(sparse_matrix: Dcsr_matrix, order=None, out: DNDarray = None):
```

Comment:
Converts `Dcsr_matrix` into a dense array (`DNDarray`). The distribution is preserved.